modules/core/src/experimental/utils/gpu-grid-aggregation/grid-aggregation-utils.js

```diff
@@ -36,7 +49,7 @@ export function pointToDensityGridData({
     worldOrigin = [-width / 2, -height / 2]; // Origin used to define grid cell boundaries
```

Comment:
This seems arbitrary. Why not just [0, 0]?

Reply:
When doing the aggregation in world space, data's bounding box's min value (bottom-left) is moved to [0, 0], in this case it is [-width/2 , -height/2], when aggregation data is retrieved, this value will be added back to get actual cell locations.